dGame/dCinema/Recorder.cpp

```diff
@@ -0,0 +1,1195 @@
+#include "Recorder.h"
+
+#include "ControllablePhysicsComponent.h"
+#include "GameMessages.h"
+#include "InventoryComponent.h"
+#include "../dWorldServer/ObjectIDManager.h"
+#include "ChatPackets.h"
+#include "EntityManager.h"
+#include "EntityInfo.h"
+#include "ServerPreconditions.hpp"
+#include "MovementAIComponent.h"
+#include "BaseCombatAIComponent.h"
+
+using namespace Cinema::Recording;
+
+std::unordered_map<LWOOBJID, Recorder*> m_Recorders = {};
+
+std::unordered_map<int32_t, std::string> m_EffectAnimations = {};
```

Comment:
These should be in an anonymous namespace as well, especially for globals so if you type m_Recorders, you are guaranteed to be referring to this m_Recorders here and not one elsewhere